app/models/repository.rb

```diff
@@ -161,14 +170,18 @@ def latest_changeset
   # Default behaviour is to search in cached changesets
   def latest_changesets(path, _rev, limit = 10)
     if path.blank?
-      changesets.find(:all, include: :user,
-                            order: "#{Changeset.table_name}.committed_on DESC, #{Changeset.table_name}.id DESC",
-                            limit: limit)
+      changesets.find(:all,
+                      include: :user,
+                      order: "#{Changeset.table_name}.committed_on DESC, "\
+                             "#{Changeset.table_name}.id DESC",
+                      limit: limit)
```

Comment:
@oliverguenther can you please rewrite those to be more rails4 compatible? Otherwise, merging with feature/rails4 will be harder.